### ISSUE TITLE
Is it possible to get the performance report after the training?

### ISSUE THREAD

**Issue by Xiaoping777** [google/model_search#23]
just wondering, after trying a number of models , ie.200 models, is it possible to output a list / file to state each model's performance / locations and etc

**Comment by hadehuang** [comment]
The same requirement, top up.

**Comment by Xiaoping777** [comment]
> The same requirement, top up.

I managed to check the results from Tensorboard, but still want performance report as output

**Comment by jayCool** [comment]
@Xiaoping777 , do you know how to do inference from one of the models were trained?

**Comment by Xiaoping777** [comment]
No, I wish the author will post the sample code. I have done some search on the web, but unfortunately did found something similar.

**Comment by anaptoro** [comment]
@Xiaoping777 How did you open it using Tensorboard?

**Comment by Xiaoping777** [comment]
> How did you open it using Tensorboard?


you need to run the tensorboard in backgroud:
1) cd tensorflow folder: /path.../.local/lib/python3.6/site-packages/tensorboard
2) python3 -m tensorboard.main --logdir=/tmp/run_example/example/ --host='host url' --port=xxxx

logdir is the folder program save log info
